File: code_examples/5-Holding_Data/12-Nested_If_Statements/CSharp/main.cs
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program uses a nested if statement
*/

using System;

/*
 * The Program class
*/
class Program {
    static void Main() {
        // this function uses a nested if statement

        int cookiesSold;

        // input
        Console.Write("Enter the number of boxes of cookies you sold: ");
        cookiesSold = Convert.ToInt32(Console.ReadLine());
        Console.WriteLine();

        // process & output
        if (cookiesSold >= 20) {
            if (cookiesSold < 30) {
                Console.WriteLine("You get a small prize.");
            } else {
                Console.WriteLine("You get a large prize.");
            }
        } else {
            Console.WriteLine("No prize.");
        }

        Console.WriteLine("\nDone.");
    }
}

File: code_examples/5-Holding_Data/12-Nested_If_Statements/JavaScript/main.js
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program uses a nested if statement
 */

const prompt = require("prompt-sync")()

// input
const cookiesSoldStr = prompt("Enter the number of boxes of cookies you sold: ")
console.log("")

// process & output
const cookiesSold = parseInt(cookiesSoldStr)

if (cookiesSold >= 20) {
  if (cookiesSold < 30) {
    console.log("You get a small prize.")
  } else {
    console.log("You get a large prize.")
  }
} else {
  console.log("No prize.")
}

console.log("\nDone.")
